packages/jazz-tools/src/tools/subscribe/SubscriptionScope.ts

```diff
@@ -69,7 +70,6 @@ export class SubscriptionScope<D extends CoValue> {
 
         if (skipRetry && value === "unavailable") {
           this.handleUpdate(value);
-          this.destroy();
```

Comment:
We don't want to destroy the subscription scope in this case?

Reply:
Yeah, the destroy shouldn't be there in first place.